### DISCUSSION TITLE
Webpack 5 + petite-vue + sfc?

### DISCUSSION THREAD

**Discussion by ricky11** [vuejs/petite-vue#105] [Q&A]
I would like to create a minimal single page application with a bundler .

Can i use petite-vue with SFC .vue files, or do i then need to add the template compiler?

**Comment by niutech** [comment]
No, Petite Vue does not use .vue single file components, use the full Vue.js instead.

That said, you can include a Petite Vue component as a single .js module like this example:
```
export default function Counter(props) {
  return {
    $template: `<div class="counter">
        <p>{{ count }}</p>
        <button @click="inc">increment</button>
      </div>`,
    count: props.initialCount,
    inc() {
      this.count++
    }
  }
}
```
